conda-recipe/bld.bat

```diff
@@ -1,11 +1,9 @@
-call "%ONEAPI_ROOT%\compiler\latest\env\vars.bat"
-if errorlevel 1 (
-    echo "oneAPI compiler activation failed"
-    exit /b 1
-)
+
+REM A workaround for activate-dpcpp.bat issue to be addressed in 2021.4
+set "LIB=%CONDA_PREFIX%\Library\lib;%CONDA_PREFIX%\compiler\lib;%LIB%"
```

Comment:
I should add that the bug is only for trying to use dpcpp in a virtual environment. The line above is within the following if statement in activate-dpcpp.bat:
    IF NOT "%CONDA_BUILD%" == ""
This line will always be true when conda-build is building the recipe using bld.bat. However if you have done something like 'conda create -n dpcpp_test dpcpp_win-64 vs2017_win-64 -c intel -c defaults --override-channels' and activate the environment, this line will be needed to be executed in order to successfully compile source code using the dpcpp executable